Authentication/LoginProviders/Facebook/FacebookLoginProvider.swift

```diff
@@ -64,6 +64,12 @@ public struct FacebookLoginProviderUser: LoginProviderUser {
         self.userId = userId
         self.accessToken = accessToken
     }
+
+    public func logOut() {
+        let loginManager = LoginManager()
```

Comment:
does this perform a facebook logout? Or just a logout in the client? I mean, if we have a facebook button, will this change to `sign in` after this logout?